### PR TITLE
More pages and general site layout

### PR DESCRIPTION
**Pull request by margaretchan**
Primary Reviewer: Sam
Secondary Reviewer: Derrill

### CODE REVIEW

**Comment by margaretchan** [comment]
Turns out I can only make 1 pull request at a time per branch. I will make new branches in the future to get around this, sorry for the large CR this time.

**Comment by margaretchan** [comment]
UI screenshots to go with commit 2eaeb8c (links turn different color when hovered): 
ID# EUe4dCP20hb and aj82YeM4HK1

File: portfolio/src/main/webapp/about.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    </header>
+    
+    <hr>
+
+    <body>
+        <div class="page-title title-margin"><h1>About Me</h1></div>
+
+        <div class="row-align center about-container reg-text-size">
+            <div class="about-bio align-left margin-side-50"><p>
+                Hello and welcome! Margaret is a rising junior at Cornell who loves climbing peices of plastic bolted
```
Comment (LiRq95) [inline portfolio/src/main/webapp/about.html:29]: Typo: Pieces

File: portfolio/src/main/webapp/about.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <hr>
+
+    <body>
+        <div class="page-title title-margin"><h1>About Me</h1></div>
+
+        <div class="row-align center about-container reg-text-size">
+            <div class="about-bio align-left margin-side-50"><p>
+                Hello and welcome! Margaret is a rising junior at Cornell who loves climbing peices of plastic bolted 
+                to walls (some call this rock climbing). When she's not doing that, you can probably find 
+                her playing tennis, walking her dog or, if its this Saturday, watching the rocket launch!
```
Comment (LiRq95) [inline portfolio/src/main/webapp/about.html:31]: its -> it's

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (margaretchan) [reply portfolio/src/main/webapp/about.html:29]: fixed

File: portfolio/src/main/webapp/about.html (hunk 2 above)
Comment (margaretchan) [reply portfolio/src/main/webapp/about.html:31]: fixed

**Pull request merged**